app/forms/apo_form.rb

```diff
@@ -101,22 +101,16 @@ def options_for_use_license_type(current_value)
     end.compact # the options block will produce nils for unused deprecated entries, compact will get rid of them
   end
 
-  def default_object_rights
-    return default_rights if model.nil?
-
-    check_default_object_rights || default_rights
-  end
-
-  def check_default_object_rights
```

Comment:
My assessment is that this was split into 2 methods to avoid a rubocop complaint about complexity.   However, my head just about broke trying to figure out if "check_xxx" should return a boolean or a string, and combining the methods resulted in fewer lines and cleaner code, IMO.

Reply:
To my mind, this change makes it easier to figure out what's doing what in this class. Thank you!